### PR TITLE
[UIK-2645][pagination] separator for total pages

### PR DESCRIPTION
**Pull request by ilyabrower**
## How has this been tested?
Manually + I've updated unit test

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [X] Nice improve.

## Checklist:

- [X] My code follows the code style of this project.
- [X] I have updated the documentation accordingly or it's not required.
- [X] Unit tests are not broken.
- [X] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [X] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

File: stories/components/pagination/__stories__/components/locales.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Pagination from '@semcore/pagination';
+import { Flex } from '@semcore/flex-box';
+
+const Demo = () => {
+  
+  return (
+    <Flex direction='column'>
+      <Pagination
+  totalPages={17824}
```
Comment (ksimanov) [inline stories/components/pagination/__stories__/components/locales.tsx:10]: Here something happened to indentation.

File: stories/components/pagination/__stories__/components/total_pages_is_1.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Pagination from '@semcore/pagination';
+import { Flex } from '@semcore/flex-box';
+
+const Demo = () => {
+  
+  return (
+    <Flex direction='column'>
+      <Pagination
+  currentPage={1}
```
Comment (ksimanov) [inline stories/components/pagination/__stories__/components/total_pages_is_1.tsx:10]: Here something happened to indentation.

**Review by ksimanov — CHANGES_REQUESTED** [review]

**Review by Valeria-Zimnitskaya — APPROVED** [review]

File: stories/components/pagination/__stories__/components/locales.tsx (hunk 1 above)
Comment (ilyabrower) [reply stories/components/pagination/__stories__/components/locales.tsx]: not mine, fixed

File: stories/components/pagination/__stories__/components/total_pages_is_1.tsx (hunk 2 above)
Comment (ilyabrower) [reply stories/components/pagination/__stories__/components/total_pages_is_1.tsx]: the same, fixed

**Review by ksimanov — APPROVED** [review]

**Pull request merged**